lightningd/channel_control.c

```diff
@@ -945,6 +946,13 @@ bool channel_on_channel_ready(struct channel *channel,
 	}
 	update_per_commit_point(channel, next_per_commitment_point);
 
+	/* FIXME: we should apply this even if it changed! */
```

Comment:
Well, we're allowed to use any alias they give us.  It would make sense to use the latest one, however, in case they decided to refresh it.  We don't, however, as I didn't want to change that behavior (without testing, at least).